10_Examples/jett-onboarding/src/talking.h

```diff
@@ -0,0 +1,23 @@
+#ifndef TALKING_H_
+#define TALKING_H_
+
+#include <iostream>
+#include "string.h"
+
+using namespace std;
```

Comment:
One important rule for C/C++ programmers...
You should _never_ put a using declaration in a header file. The reason is, we expect other people to include our header files, so any `using` statements will propogate to anything that includes it. That means everyone who uses your code is now unwillingly using everything in std. That is very problematic if they have their own version of `string` for example (or anything under std::)

Reply:
Along these same lines, any code you see in our repo is almost always defined within a namespace so that its totally clear what we are referencing.

So in this case, we would probably put most of the code in this file in between the following two lines:

```
namespace jett-onboarding{

...

// All your code
...
} // namespace jett-onboarding
```